src/backend/shortcuts/shortcuts/shortcuts.ts

```diff
@@ -175,7 +175,7 @@ async function generateMacOsApp(gameInfo: GameInfo) {
   logInfo('Generating macOS shortcut', LogPrefix.Backend)
 
   // shortcutFiles => [desktop, menu] on mac, we don't add desktop shortcut
-  const appShortcut = shortcutFiles(gameInfo.title)[1]!
+  const appShortcut = shortcutFiles(gameInfo.title)[1]
```

Comment:
this change seems unrelated

Reply:
it kind of is, but it wouldn't let me push without fixing it. it was an elinter error or something.

Reply:
oh I see, it was already removed in the main branch, looks like the main branch in your repo might be out of sync

you can probably rebase this branch on over the latest main and this should disappear